sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -29,8 +29,12 @@ serde.workspace = true
 serde_amqp = { workspace = true, optional = true }
 serde_bytes = { workspace = true, optional = true }
 tokio.workspace = true
+tokio-socks = { version = "0.5", optional = true }
+native-tls = { version = "0.2", optional = true }
+tokio-native-tls = { version = "0.3", optional = true }
```

Comment:
2 of these don't need to be referenced directly as we already have dependencies. The other has to be defined in the workspace `/Cargo.toml`. We don't allow direct dependencies unless @LarryOsterman thinks is worth exempting a version dependency in our appropriate place. This is to avoid version mismatches throughout the workspace - especially import as the number of crates grows.

Reply:
changed